src/workloads/query/ArrayTraversal.yml

```diff
@@ -228,8 +228,8 @@ AutoRun:
       mongodb_setup:
         $eq:
           - replica
-          - standalone-sbe
-          - standalone-classic-query-engine
+          - replica-query-engine-sbe
+          - replica-query-engine-classic
```

Comment:
Add `replica-all-feature-flags`?

Reply:
I'll add it, but I don't think this test ever ran on `all-feature-flags`.